### PR TITLE
[Kotlin Migration] FileUtil

### PR DESCRIPTION
**Pull request by codingtosh**
[Kotlin Migration] FileUtil

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/utils/FileUtil.kt [hunk 1]
```diff
@@ -51,45 +46,41 @@ public class FileUtil {
...
      * @param contentResolver   this is needed to open an inputStream on the content uri.
-     * @return  the internal file after copying the data across.
+     * @return the internal file after copying the data across.
      * @throws IOException
      */
-    @NonNull
-    public static File internalizeUri(Uri uri, File internalFile, ContentResolver contentResolver) throws IOException {
+    @JvmStatic
+    @Throws(IOException::class)
+    fun internalizeUri(uri: Uri?, internalFile: File, contentResolver: ContentResolver): File {
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/utils/FileUtil.kt:54]: `@KotlinCleanup("nonnull uri"))`

**Review by david-allison-1 — APPROVED** [review]
LGTM

File: AnkiDroid/src/main/java/com/ichi2/utils/FileUtil.kt (hunk 1 above)
Comment (mikehardy) [reply AnkiDroid/src/main/java/com/ichi2/utils/FileUtil.kt:54]: @david-allison-1 Does this mean you would like the annotation added prior to merge? It's my only reservation here

Comment (david-allison-1) [reply AnkiDroid/src/main/java/com/ichi2/utils/FileUtil.kt:54]: Implementer's choice - flagging it as an option to use for future migrations which flag refactors in a non-invasive way.

Fine to see it in with or without

@codingtosh

**Pull request merged**